project/cmake/scripts/linux/Install.cmake

```diff
@@ -151,33 +323,18 @@ foreach(subdir ${build_dirs})
   if(NOT subdir MATCHES kodi-platform)
     string(REPLACE " " ";" subdir ${subdir})
     list(GET subdir 0 id)
-    install(CODE "execute_process(COMMAND make -C ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/${id}/src/${id}-build install DESTDIR=${DESTDIR})")
+    install(CODE "execute_process(COMMAND ${CMAKE_MAKE_PROGRAM} -C ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/${id}/src/${id}-build install DESTDIR=${DESTDIR})")
```

Comment:
because 'make' on freebsd is not gnu make. and the generated makefiles need gnu make, aka 'gmake'.